Activities/GameActivity.cpp

```diff
@@ -372,6 +372,35 @@ void GameActivity::SetCPUTeam(int team)
 */
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+bool GameActivity::IsBuyGUIOpen(int which) {
```

Comment:
You're not using this for anything so no point in keeping it.

Reply:
This was added to compliment the `IsVisible` method from the player class as the two function similarly, and more easily expose it through a `GameActivity`. Just so I understand, why not add it for future use and to maintain consistency between the two similar methods?

Reply:
What player class? Elaborate.

Reply:
That was a mistake, I was thinking of how each player in an activity has a BuyMenuGui. I meant to refer to each BuyMenuGUI having an IsVisible() and IsOpen() method.